NHibernate.Spatial.MySQL/packages.config

```diff
@@ -1,9 +1,12 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <packages>
-  <package id="GeoAPI" version="1.7.4" targetFramework="net40-client" />
-  <package id="Iesi.Collections" version="4.0.1.4000" targetFramework="net40-Client" />
-  <package id="MySql.Data" version="6.9.9" targetFramework="net40-client" />
-  <package id="NetTopologySuite" version="1.14" targetFramework="net40-client" />
-  <package id="NetTopologySuite.IO" version="1.14.0.1" targetFramework="net40-client" />
-  <package id="NHibernate" version="4.1.1.4000" targetFramework="net40-client" allowedVersions="[4,5)" />
+  <package id="Antlr3.Runtime" version="3.5.1" targetFramework="net461" />
+  <package id="GeoAPI" version="1.7.4" targetFramework="net461" />
+  <package id="Iesi.Collections" version="4.0.2" targetFramework="net461" />
+  <package id="MySql.Data" version="6.9.10" targetFramework="net461" />

```

Comment:
I have upgraded only NHibernate package and its dependencies, and kept previous version for other packages, excepted for `MySql.Data`: previous version 6.9.9 was not seemingly available anymore, so I have taken the 6.9.10.